pypesto/ensemble/ensemble.py

```diff
@@ -219,10 +275,19 @@ def _compute_summary(tmp_array, percentiles_list):
             # create a temporary array with all the outputs needed and wanted
             tmp_output = _stack_outputs(i_cond)
             tmp_output_sensi = _stack_outputs_sensi(i_cond)
+            tmp_weights = np.ones(tmp_output.shape[-1])
+            if weighting:
+                tmp_weights = np.exp(_stack_weights(i_cond))
```

Comment:
Add a comment to explain why `np.exp` is used here (log-likelihood to likelihood?). If the method is described elsewhere, can reference